docs/api-documentation/index.html

```diff
@@ -71,16 +71,15 @@ <h1 class="govuk-heading-m">Releases</h1>
   <ul class="govuk-list govuk-list--bullet">
     <li>
       <!-- UPDATE ME to upcoming latest version -->
-      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.5/">
-        Version 2.5 - Due for release to Sandbox on 17/09/2020. Due for release to Production on 24/09/2020.
+      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.6/">
+        Version 2.6 - Due for release to Sandbox on 01/10/2020. Due for release to Production on 08/10/2020.
       </a>
     </li>
-    <li>
-      <!-- UPDATE ME to upcoming stable version if required -->
-      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.25.5/">
-        Version 1.25.5 - Due for release to Sandbox on 17/09/2020. Due for release to Production on 24/09/2020.
+    <!-- <li>
+      UPDATE ME to upcoming stable version if required
+      <a class="govuk-link" href="">
```

Comment:
Might be good to keep in some placeholder text here even though it's commented out so it's less likely to be forgotten when we're making updates? 

```suggestion
      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.25.6/">
        Version 1.25.6 - Due for release to Sandbox on xx/xx/2020. Due for release to Production on xx/xx/2020.
```